tests/sentry/monitors/test_monitor_consumer.py

```diff
@@ -137,11 +135,10 @@ def test_passing(self) -> None:
         )
 
         # Process another check-in to verify we set an expected time for the next check-in
-        expected_time = monitor_environment.next_checkin
         self.send_checkin(monitor.slug)
         checkin = MonitorCheckIn.objects.get(guid=self.guid)
         # the expected time should not include the margin of 5 minutes
-        assert checkin.expected_time == expected_time - timedelta(minutes=5)
+        assert checkin.expected_time == monitor_environment.next_checkin
```

Comment:
This is confusing me, why would a checkin that has just been processed have an expected time that is equal to the next_checkin time of the monitor?

Reply:
because the value doesn't get updated unless we re-query the DB. so the `next_checkin` is actually the expected time of this check-in. can add a comment